manabi/token.py

```diff
@@ -149,6 +149,10 @@ def _decode(
         f = key
     else:
         f = Branca(key)
-    tpb, token_payload = umsgpack.unpackb(f.decode(ciphertext, ttl))
+    try:
+        tpb, token_payload = umsgpack.unpackb(f.decode(ciphertext, ttl))
+    except Exception as e:
+        # Handle errors from msgback by creating a invalid token
+        raise RuntimeError(*e.args)
```

Comment:
I use RuntimeError to mean: The token is invalid, either because it can't be base62 decoded, the branca format is bad or now msgpack is bad. It the same thing we often do in other projects, where ValueError is used as a stand-in for a class of errors. manabi is at a point of cleanness where I should introduce custom error classes. Hmm......

Reply:
```python
try:
    # some code that might raise an exception
except Exception as e:
    raise NewError("An error occurred") from e
```

ChatGPT just told me that. I didn't know.